app/assets/javascripts/calculator_constants.js

```diff
@@ -0,0 +1,54 @@
+const CONSTANT_DESCRIPTIONS = {
```

Comment:
why not put these in the database table?

Reply:
Was waiting on feedback on where those descriptions were coming from. It's in the DB now.